src/app/components/forms/login-form/login-form.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../../styles/colors';
+
+
+.login-form-button-container {
+  text-align: center;
```

Comment:
Properties should be ordered margin-top, text-align

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)